libs/auth-api-lib/src/lib/delegations/models/delegation.model.ts

```diff
@@ -80,6 +80,12 @@ export class Delegation extends Model<
   @ForeignKey(() => Domain)
   domainName!: CreationOptional<string>
 
+  @Column({
+    type: DataType.STRING,
+    allowNull: true,
+  })
+  referenceId?: string
```

Comment:
We should add comment how this field is used. And as this is the source DB model, I think it suffice doing it here.